### ISSUE TITLE
Cannot set path for export in preferece

### ISSUE THREAD

**Issue by walkinverse** [retorquere/zotero-better-bibtex#2900]
### Debug log ID

CPY2JNWV-refs-euc/6.7.203-7

### What happened?

Re-installed the plugin as it wasn't working and I have no place to set the export directory of the library.  No place to check the cache or re-gen the cache.  It's as if the entire section of the screen went missing.  On a MacBook.  The option was in the preferences, not its gone.

<img width="860" alt="Screen Shot 2024-06-10 at 3 25 05 AM" src="https://github.com/retorquere/zotero-better-bibtex/assets/42853036/7874c1d9-5ec9-4eef-86ec-6ad04b813bda">

**Comment by retorquere** [comment]
Did you try to change the path by hand in the database perhaps?

**Comment by retorquere** [comment]
There isn't a place to modify the export directory of the library. There's something wrong with your database, I don't know what yet exactly or how it came about. If you have modified it manually I need to know.

**Comment by walkinverse** [comment]
I did not change anything manually.  I am not sure what happened.

**Comment by retorquere** [comment]
Can you cmd-q zotero, go into your zotero directory, zip up `zotero/better-bibtex.sqlite`, and attach it here?

**Comment by retorquere** [comment]
> TypeError: Zotero.ItemPaneManager is undefined. So that is 2 mac systems with the issue

That Zotero-beta needs to be upgraded

**Comment by walkinverse** [comment]
To simplify, I blew away Zotero completely from the machine Reversed to the stable release of 6, and still the same issue.  Both machines have no way to set the path for auto-export.  The error above was not from beta, it was from version 6 stable.  The attached logs are from the initial machine. 
[better-bibtex.sqlite.zip](https://github.com/user-attachments/files/15777894/better-bibtex.sqlite.zip)

This one is the other machine where I installed a fresh copy of 6.  This is the machine with the error above:

[better-bibtex.sqlite-v6-cleaninstall.zip](https://github.com/user-attachments/files/15777911/better-bibtex.sqlite-v6-cleaninstall.zip)

**Comment by retorquere** [comment]
> To simplify, I blew away Zotero completely from the machine Reversed to the stable release of 6, and still the same issue. 

Can you send a new log from that install? If the regular error report doesn't work, the Help menu will have a "send debug log to file.io" entry that can be used as a fallback.

`CPY2JNWV-refs-euc/6.7.203-7` shows [errors I haven't seen before](https://groups.google.com/g/zotero-dev/c/zTNnAaeVg1s), and these seem specific to zotero 7.

> Both machines have no way to set the path for auto-export. 

I don't know what this means TBH. You can't set a path for auto-export anywhere. Auto-export works by exporting as you usually would, and checking the "keep updated" checkbox. If you mean that you can't select a path during regular exports, something is really wrong, because that's Zotero functionality, not BBT functionality.

> The error above was not from beta,

Which error specifically?

> This one is the other machine where I installed a fresh copy of 6. This is the machine with the error above:

What error specifically?

**Comment by retorquere** [comment]
Can you try build 6427 and send a new debug log?

**Comment by retorquere** [comment]
I'd love to fix this but since I cannot reproduce this on my end, I will have to get more information from your end.

**Comment by walkinverse** [comment]
Sorry for not replying sooner.  I ended up deleting the beta version off the machine and reinstalling the stable release and that seems to clear things up and it's working again.  It's something with the beta version.  and it did it on both machines.  But since I deleted it completely and reinstalled only stable, it works!

<img width="978" alt="Screen Shot 2024-06-13 at 8 25 10 PM" src="https://github.com/retorquere/zotero-better-bibtex/assets/42853036/492358bf-d9a2-489c-b9cb-73fa449414a7">
